source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,40 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+    template <typename Policy, typename InputIt, typename Size, typename OutputIt,
+        typename ValueType>
+    OutputIt
+    histogram(Policy&& exec, InputIt start, InputIt end, Size num_bins,
+        ValueType first_bin_min_val, ValueType last_bin_max_val, OutputIt histogram_first); // (1)
+
+    template <typename Policy, typename InputIt1, typename InputIt2, typename OutputIt>
+    OutputIt
+    histogram(Policy&& exec, InputIt1 start, InputIt1 end, InputIt2 boundary_start,
+              InputIt2 boundary_end, OutputIt histogram_first);                             // (2)
+
+:code:`oneapi::dpl::histogram` computes the histogram over the data in :code:`[start, end)`
+by counting the number of elements that map to each of a set of bins and storing the counts into
+:code:`[histogram_first, histogram_first + num_bins)`. Input values that do not map to a defined
+bin are skipped silently. Sufficient output data to store each count must be provided, and the
+value type of :code:`OutputIt` must be an integral type sufficient to store the counts of the
+histogram without overflow.
+
+1. The elements of :code:`[start, end)` are mapped into bins which are evenly divided between
```

Comment:
> bins which are evenly divided between...

Should it be `bins that evenly divide (the range)` or maybe `bins which are evenly distributed between...`? Or am I nitpicking? :)

Reply:
No problem in nitpicking.  I agree that the language you propose here is better.  I've updated it.